### PR TITLE
update both api and tests

### PR DESCRIPTION
**Pull request by VentusXu09**
#187

### CODE REVIEW

**Comment by VentusXu09** [comment]
See the wiki page for details
https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/RadioReceiver-API-documentation

File: mercury/tests/test_measurement.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+def fake_event(event_uuid):
+    """
+        Mock a dummy AGEvent model
+    """
+    return AGEvent(
+        event_uuid=event_uuid,
+        event_name="fake event",
+        event_description="fake event",
+        event_date=datetime.datetime(2020, 2, 2, 20, 21, 22),
+        event_location="nyu",
```
Comment (jackxujh) [inline mercury/tests/test_measurement.py:19]: I think `mercury.models` is not going to be used soon, and we are transferring to `ag_data.models.AGEvent`, could you make a change to this?

Comment (VentusXu09) [reply mercury/tests/test_measurement.py:19]: Yes, I can do it. Please give me some time.

Comment (jackxujh) [reply mercury/tests/test_measurement.py:19]: np

Comment (VentusXu09) [reply mercury/tests/test_measurement.py]: I updated it and it passed all tests.

**Pull request merged**